Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -292,6 +292,24 @@ public static class Defaults
                 ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
                 Statistic = Statistic.Maximum,
                 Namespace = AwsNamespace.Rds
+            },
+            new AlarmDefinition
+            {
+                Name = "CPUSurplusCreditsChargedHigh",
+                Metric = "CPUSurplusCreditsCharged",
+                Enabled = false,
+                Period = TimeSpan.FromMinutes(5),
+                EvaluationPeriods = 1,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 0
+                },
+                DimensionNames = new[] { "DBInstanceIdentifier" },
+                ComparisonOperator = ComparisonOperator.GreaterThanThreshold,
+                Statistic = Statistic.Maximum,
+                Namespace = AwsNamespace.Rds,
+                TreatMissingData = TreatMissingDataConstants.Missing
```

Comment:
Do you really want TreatMissingData as missing? Are values published regularly even when zero for this metric type?

Reply:
Zero metrics are published, so I changed it to `Breaching`.